build-system/compile/bundles.config.extensions.json

```diff
@@ -508,6 +508,16 @@
       "hasCss": true
     }
   },
+  {
+    "name": "amp-image-slider",
+    "version": "1.0",
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true,
```

Comment:
Ahh, I'm sorry, we do need `amp-image-slider.css` after all with these contents:
```
/*
 * Pre-upgrade:
 * - display:block element
 * - size-defined element
 */
amp-image-slider {
  display: block;
  overflow: hidden;
  position: relative;
}

/* Pre-upgrade: size-defining element - hide children. */
amp-image-slider(.i-amphtml-built)
  > :not([placeholder]):not([slot='i-amphtml-svc']) {
  display: none;
  content-visibility: hidden;
}
```